ui/src/app/streams/stream-deploy/stream-deploy.component.ts

```diff
@@ -109,8 +109,10 @@ export class StreamDeployComponent implements OnInit, OnDestroy {
           .pipe(map((deploymentInfo) => {
             const properties = [];
             const ignoreProperties = [];
-            const cleanValue = (v) => (v && v.length > 1 && v.startsWith('"') && v.endsWith('"'))
-              ? v.substring(1, v.length - 1) : v;
+            const cleanValue = (v) => {
```

Comment:
Verified that it works. However, could you maybe refactor this into a dedicated utility method and add a few jasmine tests to that utility method? Just want to make sure that this has some basic documentation. Otherwise, nobody knows what this is doing 3 months from now :-)